observability/cockpit/how-to/configure-alerts-for-scw-products.mdx

```diff
@@ -0,0 +1,53 @@
+---
+
+This page shows you how to configure alerts for Scaleway resources using the [Mimir](/observability/cockpit/concepts/#mimir) alert manager. As Grafana's alert manager does not run indefinitely, we recommend using Mimir's alert manager to receive alerts on your resources.
+
+<Message type="requirement">
+  - You have an account and are logged into the [Scaleway console](https://console.scaleway.com/)
+  - You have [activated your Cockpit](/observability/cockpit/how-to/activate-cockpit/)
+  - You have created Scaleway resources to monitor
+  - You have [created your Grafana credentials](/observability/cockpit/how-to/retrieve-grafana-credentials/) and selected the **Editor** role
+  - You have [activated managed alerts and created at least one contact point](/observability/cockpit/how-to/activate-managed-alerts/)
+</Message>
+
+1. Click **Cockpit** in the Observability section of the [console](https://console.scaleway.com/) side menu. The **Cockpit** overview page displays.
+2. Click **Open dashboards** at the top right corner of your screen. You are directed to Grafana.
+3. Log in to Grafana using your credentials.
+4. Click the **Toggle menu** then click **Alerting**.
+5. Click **Alert rules** and **+ Create alert rule**.
+6. Enter a name for your alert.
+7. Select **Mimir or Loki alert** and the data source you want to configure alerts for. For the sake of this documentation we are choosing the **Scaleway Metrics** datasource.
+8. In the **Metrics browser** drop-down, select the metric you want to configure an alert for. For the sake of this documentation, we are choosing the `instance_server_cpu_seconds_total` metric.
+9. Select labels that apply to the metric you have selected in the previous step, to target your desired resources and fine-tune your alert.
+10. Select one or more values fo your labels.
+11. Click **Use query** to generate your alert based on the previous information you have selected. For example, the alert below will be triggered whenever **the rate of our Instance's total CPU usage per second exceeds 10%**.
+    <Lightbox src="scaleway-instance-grafana-alert.webp" alt="" />
+12. In the **Alert evaluation behavior** field, configure the amount of time you want the alert to be firing for.
+    <Message type="note">
+     For example, if you wish to be alerted after your alert has been firing for 2 minutes without any interruption, type `2` and select `minutes` in the drop-down.
+    </Message>
+13. Enter a namespace for your alert in the **Namespace** field and click **Enter**.
+14. Enter a name for your alert's group in the **Group** field and click **Enter**.
+15. Optionally, add a summary and a description.
+16. Click **Save** at the top right corner of your screen to save your alert. Once your alert meets the requirements you have configured, you will receive an email to inform you that your alert is firing.
+
+<Message type="important">
+ You can configure up to a maximum of 10 alerts for your resources.
```

Comment:
suggestion: You can configure up to a maximum of 10 alerts for the "Scaleway Metrics" data source.